lib/maplibre-compose/src/maplibreNativeMain/kotlin/dev/sargunv/maplibrecompose/compose/offline/DownloadProgress.kt

```diff
@@ -0,0 +1,19 @@
+package dev.sargunv.maplibrecompose.compose.offline
+
+public sealed interface DownloadProgress {
+  public data object Unknown : DownloadProgress
+
+  public data class Normal(
+    val completedResourceCount: Long,
+    val completedResourceBytes: Long,
+    val completedTileCount: Long,
+    val completedTileBytes: Long,
+    val downloadState: DownloadState,
+    val isRequiredResourceCountPrecise: Boolean,
+    val requiredResourceCount: Long,
```

Comment:
I wonder why there is no `requiredTileCount`. `resources` probably refer to fonts, icons and such?

Reply:
I'm unsure of the difference between tile and resource here. These are 1:1 with what the MLN provides

Reply:
If I had to guess, I'd say resource count is tile count plus other resources